store/tikv/coprocessor.go

```diff
@@ -484,7 +495,24 @@ const minLogCopTaskTime = 300 * time.Millisecond
 // send the result back.
 func (worker *copIteratorWorker) run(ctx context.Context) {
 	defer worker.wg.Done()
+
 	for task := range worker.taskCh {
+		worker.actionOnExceed.mu.Lock()
+		if worker.actionOnExceed.mu.exceeded != 0 {
+			worker.actionOnExceed.mu.aliveWorker--
+			logutil.BgLogger().Info("end one copIterator worker.",
```

Comment:
Better to move log out of the lock section.

Reply:
updated.